FlightReservation/src/components/form/GoogleButton.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import {View, Text, Pressable} from 'react-native';
+import style from '../../consts/style';
+
+const GoogleButton = ({text, onPress}) => {
+    return (
+        <Pressable
+        onPress={onPress}>
+            <View style={style.btn}>
+                <Text style={style.btnForm}>
+                    {text}
+                </Text>
+            </View>
+        </Pressable>
+    );
+};
+
+export default GoogleButton;
```

Comment:
is a general propose button, is well implemented but find a way that it could be reused for any function, you already have general button pleas try to find a way to re-use that button